### PR TITLE
Designer workspace ops#391

### PR DESCRIPTION
**Pull request by cas--**
**Create a designer role page**

Add a new page detailing the role of a designer with workflow, tools, etc.

Refs: niteoweb/operations#391

**Additional commit created the `roles` folder.**

### CODE REVIEW

File: roles/designer.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Communication
+
+### Slack channel
+
+There is a dedicated channel `#design` for designers to discuss relevant work.
+
+### Sharing Files
+
+We use Resilio to share files between people and there is a dedicated `Design` folder for designers to use. Ideally, this should only be for those larger files that cannot be attached to User Stories.
```
Comment (cas--) [inline roles/designer.md]: Needs a llink to resilio page

File: roles/designer.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Use sub-folders to group related files. It should be named using a short two or three-word description (separated by hyphens), along with the associated GitHub repository and issue number e.g. `niteo.co-design-ops#401`
```
Comment (cas--) [inline roles/designer.md]:
```diff
-related files
+related project files
```

**Comment by yevnik** [comment]
To this one `We use Resilio to share files between people and there is a dedicated Design folder for designers to use. Ideally, this should only be for those larger files that cannot be attached to User Stories.`
I would add that all final source files should be shared (like PSD, AI, PDF, HTML structures, etc)

**Comment by alexakinyeye** [comment]
Looks good to me.

**Pull request merged**